operators/vision/draw_bounding_box.hpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "ocos.h"
+#include "string_utils.h"
+
+namespace ort_extensions {
+
+enum class BoundingBoxFormat {
+  XYWH,
+  XYXY,
+  Center_XYWH,
+};
```

Comment:
Can we link to some documentation both here and in the Step explaining exactly what these formats are so that a new user who is not familiar with bounding boxes can more easily understand? The Keras documentation would be fine if that's the best we can find.

Reply:
Sure.